bundles/core/org.eclipse.smarthome.core/ESH-INF/config/networkinterface.xml

```diff
@@ -7,7 +7,7 @@
 	<config-description uri="system:network">
 		<parameter name="primaryAddress" type="text">
 			<label>Primary Address</label>
-			<description>A subnet (e.g. 192.168.1.0/24) <!-- or an IP address (e.g. 192.168.1.5) --></description>
+			<description>An IPv4/IPv6 CIDR notation (e.g. 192.168.1.0/24 or 192.168.1.12 or fe80:0:0:0:0:0:c0a8:11%eth0/120)</description>
```

Comment:
The idea was to have the NETWORK address here and **not** the IP address directly. The reason behind it is that if the ESH installation obtains its IP address via DHCP it might get a different one from the same network and we do not want to require the user to change the configuration if this happens. That's why the original implementation in #3930 calculated the NETWORK address and stored it as a configuration value.

I think we should restore this behavior and for IPv6 we would store the subnet, not the specific address as well. Because in IPv6 a host could use auto configuration and choose a different IP from the same subnet on every restart of the system (privacy extension).

Reply:
you're right that there is an IP in the comment, that's my mistake